### PR TITLE
Update README.md

### LINKED ISSUE #4: error: No remote refs found similar to ‘flathub-beta’
Flathub's regular branch works perfect on my system but after following the readme attached to this repo I can't finish the installation of the Flatpak due to above error the result of flatpak remotes shows its installed

```
Name         Options
fedora       system,oci
flathub      system
flathub-beta system
winepak      system
```
```

           /:-------------:\          solojazz@hardware
        :-------------------::        OS: Fedora 
      :-----------/shhOHbmp---:\      Kernel: x86_64 Linux 5.1.5-300.fc30.x86_64
    /-----------omMMMNNNMMD  ---:     Uptime: 10m
   :-----------sMMMMNMNMP.    ---:    Packages: 1671
  :-----------:MMMdP-------    ---\   Shell: bash 5.0.7
 ,------------:MMMd--------    ---:   Resolution: 1920x1080
 :------------:MMMd-------    .---:   DE: GNOME 
 :----    oNMMMMMMMMMNho     .----:   WM: GNOME Shell
 :--     .+shhhMMMmhhy++   .------/   WM Theme: 
 :-    -------:MMMd--------------:    GTK Theme: Mojave-dark [GTK2/3]
 :-   --------/MMMd-------------;     Icon Theme: McMojave-circle-dark
 :-    ------/hMMMy------------:      Font: Cantarell 11
 :-- :dMNdhhdNMMNo------------;       CPU: Intel Core i7-3930K @ 12x 3.8GHz [34.0°C]
 :---:sdNMMMMNds:------------:        GPU: Radeon RX 580 Series (POLARIS10, DRM 3.30.0, 5.1.5-300.fc30.x86_64, LLVM 8.0.0)
 :------:://:-------------::          RAM: 2278MiB / 15977MiB
 :---------------------://       
```

### PR DESCRIPTION
**Pull request by LeandroStanger**
Fixes #4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,20 +9,20 @@ ___________________________________________
    1. Open the application called 'Terminal'
    2. In this application run the command:
 ```
-flatpak remote-add flathub-beta https://flathub.org/beta-repo/flathub-beta.flatpakrepo && flatpak update --appstream
+flatpak remote-add --user flathub-beta https://flathub.org/beta-repo/flathub-beta.flatpakrepo && flatpak update --appstream
```
Comment (gasinvein) [inline README.md]: Why `&&` btw? Maybe two commands instead of one-liner would be better?

**Comment by gasinvein** [comment]
>Open the application called 'Terminal'

>In this application run the command

Do we really need to be _that_ verbose? I'm pretty sure every single Linux user is aware how to run a command.

File: README.md [hunk 2]
```diff
@@ -6,23 +6,24 @@ ___________________________________________
 
 ## How install Lutris Beta
 1. Install Flathub Beta
-   1. Open the application called 'Terminal'
-   2. In this application run the command:
+
+Enter this command in a terminal window
```
Comment (gasinvein) [inline README.md]: IMO this is redundant, too. Monospace text here is obviously a command to run.

Comment (LeandroStanger) [reply README.md]: ## How install Lutris Beta
1. Install Flathub Beta

**Command:**
```
flatpak remote-add --user flathub-beta https://flathub.org/beta-repo/flathub-beta.flatpakrepo
flatpak update --appstream
```

**Pull request merged**